h5glance/terminal.py

```diff
@@ -11,6 +11,15 @@
 from subprocess import run
 import sys
 
+def fmt_dtype(dtype):
+    if isinstance(dtype, str):
```

Comment:
In what condition does this happen? I've worked with some string fields, and they seem to have `np.dtype('O')` as their dtype; I haven't come across anything where the dtype is a string.

Reply:
I now think it was a mistake/overlook from my side. I now added proper support for vlen types; I'm not confident in the choice of labels ('bstr' (the other varian was 'ASCII') and 'vlen' look somewhat too "pidgin", and 'str' might seem a bit ambiguous). Please feel free to correct.